artemis/base_pytest.py

```diff
@@ -1,5 +1,5 @@
 import os
-import json
+import ujson
```

Comment:
```suggestion
import ujson as json
```
this will reduce the diff and allow conditional import